test/tests/markers/minimum_element_size/minimum_element_size_value.i

```diff
@@ -0,0 +1,55 @@
+[Mesh]
```

Comment:
Can you remove the dots from the input file? Those are outdated and we should stop using them. We just need the square brackets now.